virttest/qemu_monitor.py

```diff
@@ -850,20 +849,28 @@ def _read_up_to_qemu_prompt(self, timeout=PROMPT_TIMEOUT):
                 pass
         return False, "\n".join(s.splitlines())
 
-    def _send(self, cmd):
+    def _send(self, cmd, fds=None):
         """
         Send a command without waiting for output.
 
         :param cmd: Command to send, type: bytes
```

Comment:
Can we add a description for the `fds` argument here? It would be consistent and add more understanding for the changes here.

Reply:
Thanks, updated!